stdlib/@python2/antigravity.pyi

```diff
@@ -1,4 +1 @@
 import sys
```

Comment:
I guess you'll have to remove the import too.

Reply:
The Python 2 stubs don't get checked for missing imports. We have this in `.flake8`:

```
exclude = .venv*,@*,.git,*_pb2.pyi
```

Is this intentional?

Reply:
Not sure what that's there for. It predates the `@python2` directory, since it was there before 9af49c0b6. I think we can just remove it.

Reply:
Looking at git history, this is not intentional; ignoring `@*` was added before the big directory restructuring.

Reply:
We should probably remove it after this PR, since I expect many new errors.

Reply:
I started cleaning unused imports from Python 2 stubs, but I'd rather have that in a separate PR because it's a lot of manual work.

Reply:
Sounds good. I think we should just merge this PR and follow up on other issues later.